modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,83 @@
+
+* Egress routers block the live migration process. This feature must be removed before beginning the live migration process.
+
+* During the live migration, multicast, egress IP addresses, and egress firewalls are temporarily disabled. They can be migrated from OpenShift SDN to OVN-Kubernetes after the live migration process has finished.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* The live migration not supported on HyperShift clusters.
+
+* New nodes should not be created during the live migration because both CNIs would try to setup the new node.
+
+* OpenShift SDN does not support IPsec. After migration, cluster administrators can enable IPsec.
+
+* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable IPv4, IPv6, or dual-stack if necessary after migration is completed.
+
+* The cluster MTU is the MTU value for pod interfaces. It is always less than your hardware MTU to account for the cluster network overlay overhead. The overhead is 100 bytes for OVN-Kubernetes and 50 bytes for OpenShift SDN.
++
+During the live migration, both OVN-Kubernetes and OpenShift SDN run in parallel. OVN-Kubernetes manages the cluster network of some nodes, while OpenShift SDN manages the cluster network of others. To ensure that cross-CNI traffic remains functional, the Cluster Network Operator updates the routable MTU to ensure that both CNIs share the same overlay MTU. As a result, after the migration has completed, the cluster MTU is 50 bytes less.
+
+* Some parameters of OVN-Kubernetes cannot be changed after installation. The following parameters can be set before starting the live migration:
+
+** `v4InternalSubnet`
+** `v4TransitSwitchSubnet`
+
+* Unless otherwise configured, OVN-Kubernetes uses subnets `100.64.0.0/16`, `100.88.0.0/16`, and `169.254.169.0/29` internally by default. Cluster administrators must ensure that these addresses do not overlap with any existing subnets in their network. If there is an overlap, administrators must explicitly set the `v4InternalSubnet` and `v4TransitSwitchSubnet` before initiating the live migration.
+
+* In most cases, the live migration is independent to the secondary interfaces of pods created by the Multus CNI plugin. However, if these secondary interfaces were set up on the default network interface controller (NIC) of the host, for example, using macvlan or bridge interfaces with the default NIC as the control node, OVN-Kubernetes might encounter malfunctions. Users should remove such configurations before proceeding with the live migration.
```

Comment:
One more caveat shall be added. When there are multiple NICs in the host, and the default route is not on the interface that has the Kubernetes NodeIP. Users shall use the offline migration instead.